server2/Arcadia.Assistant/Arcadia.Assistant.MobileBuild.Contracts/Interfaces/IMobileBuildActor.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceFabric.Actors;
+using Microsoft.ServiceFabric.Actors.Remoting.FabricTransport;
+using Microsoft.ServiceFabric.Services.Remoting;
+
+[assembly: FabricTransportActorRemotingProvider(RemotingListenerVersion = RemotingListenerVersion.V2_1, RemotingClientVersion = RemotingClientVersion.V2_1)]
+namespace Arcadia.Assistant.MobileBuild.Contracts.Interfaces
+{
+    /// <summary>
+    /// This interface defines the methods exposed by an actor.
+    /// Clients use this interface to interact with the actor that implements it.
+    /// </summary>
+    public interface IMobileBuildActor : IActor
+    {
+        /// <summary>
+        /// Return current stored build data
+        /// </summary>
+        /// <returns>Stored build data</returns>
+        Task<byte[]> GetMobileBuildDataAsync(CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Return current stored build version
+        /// </summary>
+        /// <returns>Stored build version</returns>
+        Task<string> GetMobileBuildVersionAsync(CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Update current stored build by data and version
+        /// </summary>
+        /// <param name="version">New build version</param>
+        /// <param name="data">New build data</param>
+        Task SetMobileBuildData(string version, byte[] data, CancellationToken cancellationToken);
```

Comment:
I don't think we want to be able to cancel save operation :) Or you need to support state rollback